contracts/ActiveBridgeSetLib.sol

```diff
@@ -28,10 +28,18 @@ library ActiveBridgeSetLib {
     uint256 lastBlockNumber;
   }
 
+  modifier validBlockNumber(uint256 _currentBlock, uint256 _lastBlock) {
+    require (_currentBlock >= _lastBlock, "The last updated block was higher than the one provided");
```

Comment:
Do you mind if we go the other way so the message explicitly complains about the bad argument?
```suggestion
    require (_currentBlock >= _lastBlock, "The provided block is older than the last updated block");
```